app/views/select-your-business.njk

```diff
@@ -12,9 +12,7 @@
     <form method="POST" autocomplete="off" novalidate="novalidate">
       <input type="hidden" name="crumb" value="{{crumb}}"/>
 
-      {{ govukInsetText({
-        text: "Select the business that’s claiming:"
-      }) }}
+      <p>Select the business that’s claiming:</p>
```

Comment:
Is this change covered by a ticket? I know there was one for apply.

Reply:
Yeah did not strictly see a ticket - but saw it in the prototype -> https://fcp-ahwr-prototype.herokuapp.com/v6/farmer/claim/choose-sbi

Will ask Zohaib to make a ticket retrospectively to cover testing effort